conf/v1.routes

```diff
@@ -1,11 +1,14 @@
 # microservice specific routes
 
-GET        /hello-world                                 v1.controllers.MicroserviceHelloWorldController.hello()
-
 POST       /reporting-company/appoint                   v1.controllers.AppointReportingCompanyController.appoint()
 
 POST       /reporting-company/revoke                    v1.controllers.RevokeReportingCompanyController.revoke()
 
-POST       /return/abbreviated                          v1.controllers.AbbreviatedReturnController.submitAbbreviatedReturn()
+POST       /return/abbreviated                          v1.controllers.AbbreviatedReturnController.submitAbbreviatedReturn(isInternal: Boolean = false)
+
+POST       /internal/return/abbreviated                 v1.controllers.AbbreviatedReturnController.submitAbbreviatedReturn(isInternal: Boolean = true)
+
+POST       /return/full                                 v1.controllers.FullReturnController.submit(isInternal: Boolean = false)
```

Comment:
Can't you just override this with the url /return/full?isInternal=true

Reply:
This is a great question. In the controller, since we give it a value in the controller, then it would never take into account `isInternal` parameter passed by the user, it would throw an error along the lines of "not allowed to add parameter to fixed or default value"(something like that). Not only that, since we go entirely off the route - your suggestion would result in a 404 not found as that route does not exist (even if you had something like /return/full/:isInternal) -- I hope this helps??